packages/models/src/store/store.ts

```diff
@@ -94,7 +114,7 @@ export class Store<
   }
 
   private deserializeField(type: StorageLocation, option: unknown, value: string) {
-    this.assetStorage(this.getStorage(type))
+    this.assertStorge(this.getStorage(type))
```

Comment:
`Storge` means?

Reply:
It's the method to asset `storage` is not empty. Here I just fix a typo `asset => assert` 

Ref: https://github.com/ckb-js/kuai/pull/163/files/9bc8c02c829db9dfaef273518abb4fd8b96399cc#diff-84f82286c84c4c0e853797d892f4e8a6c907a3c7298583c756b9dde3c898c8c5R325-R327

Reply:
Maybe `assertStorage`?

Reply:
> Maybe `assertStorage`?

My fault, fixed by https://github.com/ckb-js/kuai/pull/163/commits/a536ebb88185e284fcf62bb4f80b64e61cb523ba